app/src/main/java/org/p2p/wallet/auth/ui/phone/PhoneNumberEnterContract.kt

```diff
@@ -6,12 +6,19 @@ import org.p2p.wallet.common.mvp.MvpView
 
 interface PhoneNumberEnterContract {
     interface View : MvpView {
+        enum class ContinueButtonState {
```

Comment:
why not?
there can be multiple ContinueButtonState in future, and this one is connected only with this View

Reply:
why not?
there can be multiple ContinueButtonState in future, and this one is connected only with this View

Reply:
Because the `View` interface is about the functions not the state of the button.
Moving this to a separate class is especially required if there will be more states in the future. 
Let's leave the interface consistent and clean